tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -100,6 +102,122 @@ def compare(ert_obs_dict: dict, yaml_obs_dict: dict) -> None:
             assert yaml_obs[list_item["key"]][2] == ert_obs_dict[list_item["key"]][2]
 
 
+def create_schedule_from_data(
```

Comment:
This is an internal function, I suggest to use a leading underscore to empathize that. 

```suggestion
def _create_schedule_from_data(
```